ibc-apps/ics20-transfer/types/src/error.rs

```diff
@@ -17,86 +16,46 @@ pub enum TokenTransferError {
     ContextError(ContextError),
```

Comment:
Sounds like this `ContextError` variant not used anywhere.

Reply:
It's used [here](https://github.com/cosmos/ibc-rs/blob/60ff9bd16d43d3475ba443069868959f0fdc5bd6/ibc-apps/ics20-transfer/types/src/msgs/transfer.rs#L57) and [here](https://github.com/cosmos/ibc-rs/blob/60ff9bd16d43d3475ba443069868959f0fdc5bd6/ibc-apps/ics20-transfer/types/src/msgs/transfer.rs#L63) in a `TryFrom<RawMsgTransfer> for MsgTransfer` implementation.

Reply:
Was hidden from the eyes. Fine :pray:!